tests/test_classlist.py

```diff
@@ -957,3 +958,48 @@ def test_get_item(two_name_class_list):
     alice = InputAttributes(name="Alice")
     assert two_name_class_list[alice] == two_name_class_list["Alice"]
     assert two_name_class_list[alice] == two_name_class_list[0]
+
+
+class TestPydantic:
```

Comment:
Could you please add a `skipif` here for the case that pydantic is not installed, much like the one for matlabengine in `test_convert.py` (recalling the transferrability conversation we had previously)